frontend/src/admin/components/cancel-edit-icon/cancel-edit-icon.tsx

```diff
@@ -0,0 +1,9 @@
+import React from 'react'
+
+export default function CancelEditIcon() {
```

Comment:
You are naming the component based on what it used for and not on what is.
If the icon used for the cancel action is a cross, then perhaps this should be called `CrossIcon`.
Then it can be used for other things than just exiting edit mode.

The same is true for the other icons.